README.md

```diff
@@ -64,7 +64,7 @@ Fork or download the HTM-Community htm.core repository from https://github.com/h
 ```
 cd to-repository-root
 python -m pip install --user --upgrade pip setuptools setuptools-scm wheel
-python -m pip install --no-cache-dir --user -r bindings/py/packaging/requirements.txt
+python -m pip install --no-cache-dir --user -r requirements.txt
```

Comment:
moved requirements here, TODO might need move back (for pypi dist/ build)

Reply:
Yes, as I mentioned before, in order to get the requirements.txt file into the wheel it needs to somehow get into the Release/distr/dist/